site/content/docs/main/upgrade-to-1.10.md

```diff
@@ -69,16 +77,16 @@ Before upgrading, check the [Velero compatibility matrix](https://github.com/vmw
 
     ```bash
     Client:
-        Version: v1.9.0
+        Version: v1.10.0
         Git commit: <git SHA>
 
     Server:
-        Version: v1.9.0
+        Version: v1.10.0
     ```
 
 ## Notes
 ### Default backup storage location
-We have deprecated the way to indicate the default backup storage location. Previously, that was indicated according to the backup storage location name set on the velero server-side via the flag `velero server --default-backup-storage-location`. Now we configure the default backup storage location on the velero client-side. Please refer to the [About locations][9] on how to indicate which backup storage location is the default one.
+We have deprecated the way to indicate the default backup storage location. Previously, that was indicated according to the backup storage location name set on the velero server-side via the flag `velero server --default-backup-storage-location`. Now we configure the default backup storage location on the velero client-side. Please refer to the [About locations][10] on how to indicate which backup storage location is the default one.
```

Comment:
Is this still needed?  I don't think we need it in all future versions.

Reply:
I've removed it